fansided/wordpress-image-crate
# Add Imagn image provider

Requires `IMAGN_API_TOKEN` PHP constant to be defined in the `wp-config.php` file before release.

## QA Instructions
1) Open post editor on staging sites - https://fansided.staging.p104.sites.pressdns.com/wp-admin/post.php?post=1952834&action=edit

2) Click “Add Media” above text editor

3) Navigate to “Imagn Images” on left-side nav

4) Enter a search term like “nfl” and click “Search”

5) See results

6) Click on an image

7) Click “Download Imagn Image” image on the bottom right

8) See the main media library with downloaded image focused

9) Click “Insert into post” on the bottom right

10) See image inserted into post content with caption
